### PR TITLE
Update combinators table in README

### PR DESCRIPTION
**Pull request by chicoxyzzy**
Per July 2019 meeting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
-| -------------------- | ----------------------------------------------- | --------------------------------------------------------------------------- |
-| `Promise.allSettled` | does not short-circuit                          | [separate proposal](https://github.com/tc39/proposal-promise-allSettled) 🔜 |
-| `Promise.all`        | short-circuits when an input value is rejected  | added in ES2015 ✅                                                          |
-| `Promise.race`       | short-circuits when an input value is settled   | added in ES2015 ✅                                                          |
-| `Promise.any`        | short-circuits when an input value is fulfilled | this proposal 🆕                                                            |
+| name                 | description                                     |                    |
+| -------------------- | ----------------------------------------------- | ------------------ |
+| `Promise.allSettled` | does not short-circuit                          | added in ES2015 ✅ |
```
Comment (mathiasbynens) [inline README.md]: It would be scheduled for inclusion in ES2020 (but depending on when the PR actually gets merged and approved). Definitely not ES2015.

Comment (mathiasbynens) [reply README.md]: Let's still link to the proposal until this is actually merged upstream.

Comment (chicoxyzzy) [reply README.md]: Ok sure

**Comment by chicoxyzzy** [comment]
This should be merged after https://github.com/tc39/ecma262/pull/1583 will be merged

**Comment by chicoxyzzy** [comment]
https://github.com/tc39/ecma262/pull/1583 is not merged yet, but `Promise.allSettled` was added to finished proposals table in https://github.com/tc39/proposals/commit/034ebab571dd118e2b9d3131b9d588585bb2bddf

**Review by mathiasbynens — APPROVED** [review]

**Pull request merged**